Lab2.ipynb

```diff
@@ -0,0 +1,636 @@
z/4iKXQq7kRESik1Vyk25zT3FRgN9AQGmNlI39py4A9m9izwL9/z35nZLGB1bhcq7FmwIiIFUaFdO2reN55jq1Zx4NlnvY5TJgV5HUBERIpGRNxAIuIGsnPoMK+jSBbOuZnAzGxrvwA3ZPvW4fm41jxgHkBsbKz6kIuI5yKuu46Ur77iwDOzKd+yJWHdunkdqUzRnTsREZGC0dxXEREfM6PWpEmUi45m99g/c/LHH72OVKaouBMRKQM0FqFIae6riEgWAeXLEzVrJpiRMGo06cnJXkcqM1TciYiUchqLUHg091VEJH9CoqKoO306J7ZuZe+DE3FOO8eLg87ciYiUchFxA1XYFRLn3OBc1t8C3irmOCIifi2sy2+IvGMMSTOepnzr1lQdNtTrSKWe7tyJiJQRGosgIiLFrdqIEYT99rfsf/xxktet8zpOqafiTkSkDNBYBBER8YIFBFBn6hRCoqJIuOtuUvcneh2pVFNxJyJSBkTEDaT+opcJjY72OoqIiJQxgeHhRM2aSXpyMrvvuAN38qTXkUotFXciImWMOmeKiEhxK9ekCXUem8zxjRvZ/8R0r+OUWiruRETKEHXOFBERr1Tq3ZuIYUM5vGgRP7/zjtdxSiUVdyIiZUhE3EAqdOjgdQwRESmjat57L+XbtGHvhAc48cMPXscpdVTciYiUQeqcKSIiXrCQEOr+9SksKIjdd95FekqK15FKFRV3IiJljDpnioiIl4Lr1KHOE49zYssW9j3yiNdxShUVdyIiZYw6Z4qIiNfCunal2shb+Okfyzmy/HWv45QaKu5ERERERKTYRY4eTYWLL2bfww+TsuU7r+OUCiruRETKMI1FEBERr1hgIHWnP0FAeBi777iDU8eOeR2pxFNxJyJSRmksgoiIeC0oMpK6Tz7JyR9/ZO9f/oJzzutIJZqKOxGRMkpjEURExB9U7NiRyDvv5Ojb73D4lVe9jlOiqbgTESnjNBZBRES8Vu2m4YR1787+adM4/vXXXscpsVTciYiUYRqLICIi/sACAqgzdQpBkdXZffc9nDp61OtIJZKKOxGRMkxjEURExF8EVqlC3elPkrpnD/smTtT5u/NQ7MWdmVU0s/Vm1tf3/Gozm29mS8ysV3HnERERERER/1ChXVsix4zh57fe5siyZV7HKXGC8vuNZrYA6AskOudaZlnvDTwNBALPO+emnuVS44DMgx3OuRXACjOLAKYD/5f/+JCamkpCQgIpKSnn8jIpA0JDQ4mKiiI4ONjrKCIlQsbZu0p9+xIRN9DrOCIiUkZVu/kmkv/7KfsnP0aFmBjKNWnidaQSI9/FHbAQeAZ4OWPBzAKB2cDlQAKwzsze4HShNyXb628E2gDfAqE5XP8B37XOSUJCAuHh4TRo0AAzO9eXSynlnOPgwYMkJCTQsGFDr+OI+L2MsQgp8fEAKu5ERMQzFhBAnWnT2H51f3bffTcNli4loHx5r2OVCPnelumcWw0cyrbcEdjmnNvunDsJvAb0c8597Zzrm+2RCHQHLgGuA242swA7bRrwtnPu8+zva2YjfNs41yclJZ2RKyUlhWrVqqmwk18xM6pVq6Y7uiL5pLN3IiLiT4IiI6kzbRontm5j/5SzbQyUDAU9c1cX2JXleYJvLUfOuQnOuTuBV4H5zrl0YDTQExhgZiNzeM0851yscy42MjIyx+uqsJOc6L8LkfOTvG6dxiKIiIjnwn7TmWo338SRpUv5+e23vY5TInjSLdM5t9A5t9L355nOufbOuZHOuble5CmowMBAYmJiMh87duxg/fr1jBkzJt/XOHLkCHPmzMn16/v27WPQoEE0btyY9u3b06dPH7777rvCiH9WDRo04MCBA3l+z8KFC9mzZ0/m85tuuolvv/22qKOJSCHL2J6psQhFK7dmYlmbjplZdzNbY2Zzzay7h3FFRDwTOWYM5du0Ye9fHuTkrl1nf0EZV9DibjdQL8vzKN9amVK+fHk2btyY+WjQoAGxsbHMnDnzjO9NS0vL8Rp5FXfOOfr370/37t35/vvv2bBhA1OmTGH//v2F+jkKIntx9/zzz9O8eXMPE4nI+YiIG0iFDh28juHXzGyBmSWa2aZs673NbIuZbTOz8Xldwzm3wjl3MzASiMvypaxNxxxwjNPn1BMK7xOIiJQcFhxMnSefhIAAdt9zL+7kSa8j+bWCFnfrgCZm1tDMQoBBwBsFj1XyrVq1ir6+34BPmjSJoUOH0rlzZ4YOHco333xDx44diYmJoXXr1mzdupXx48fz/fffExMTw9ixY391rQ8++IDg4GBGjvzfrtU2bdrQpUsXnHOMHTuWli1b0qpVK5YsWZL5/t27d2fAgAFER0czZMgQnHO88847XHvttTnmXLx4Ma1ataJly5aMGzfujM+0Y8cOWrbMbJTK9OnTmTRpEsuWLWP9+vUMGTKEmJgYjh8/Tvfu3Vm/fn2e1w0LC2PChAm0adOGSy65xK+KVRGRPCwEemddyNJg7AqgOTDYzJqbWSszW5ntUSPLSzObiZnZ5ZxuOpbo+9oa59wVnC74HirSTyQi4sdCoupS+5FHSPnqKxJnPO11HL92LqMQFnO6IUp1M0sAJjrnXjCzUcC7nO6QucA5902RJM2Hh/71Dd/u+blQr9m8TiUmXtUiz+85fvw4MTExADRs2JDXX3/9jO/59ttvWbt2LeXLl2f06NHccccdDBkyhJMnT3Lq1CmmTp3Kpk2b2Lhx4xmv3bRpE+3bt8/xvZcvX87GjRv58ssvOXDgAB06dKBr164AfPHFF3zzzTfUqVOHzp0789FHH9GzZ09GjBjBL7/8QsWKFVmyZAmDBg1iz549jBs3jg0bNhAREUGvXr1YsWIFV1999Vl/RgMGDOCZZ55h+vTpxMbG/upreV33l19+4ZJLLmHy5Mn8+c9/Zv78+TzwwANnfT8RKXoai5A759xqM2uQbTmzwRiAmWU0GJvC6TFCv2KnDwVP5dfNxLoDFTldHB4H3vKt
```

Comment:
I don't think this gigantic zero-valued array is what you intend to dump out here.